README.md

```diff
@@ -68,6 +68,9 @@ git clone https://github.com/alibaba/inclavare-containers
 2. Install the Dependency required by Inclavare Containers
 - Go version 1.14 or higher.
 - `libseccomp`.
+- [SGX DCAP](https://github.com/intel/SGXDataCenterAttestationPrimitives): please download and install the rpm(centos) or deb(ubuntu) from "https://download.01.org/intel-sgx/sgx-dcap/#version#linux/distro"
+	- libsgx-dcap-quote-verify: both for centos and ubuntu
+	- libsgx-dcap-quote-verify-dev(ubuntu) or libsgx-dcap-quote-verify-devev(centos)
 
```

Comment:
If build rune needs `libsgx-dcap-quote-verify` and `libsgx-dcap-quote-verify-dev`, you need to install dependencies in test images. (.github/workflows/docker/Dockerfile-centos8.1 and .github/workflows/docker/Dockerfile-ubuntu18.04) . Then you need to rebuild test images. Only add compile dependence in compile check images is not enough